### PR TITLE
.gitignore and index.html created

### PR DESCRIPTION
**Pull request by eduardoccmelo**
all files updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <nav>
+            <ul>
+                <a href="#home"><li>Home</li></a>
+                <a href="#about"><li>About</li></a>
+                <a href="#history"><li>History</li></a>
+                <a href="#location"><li>Location</li></a>
+                <a href="#gastronomy"><li>Gastronomy</li></a>
+                <a href="#more"><li>More</li></a>
+                <a href="#contact"><li>Contact</li></a>
+            </ul>
```
Comment (mpagels) [inline index.html]: the li elements need to include the a href

```
<ul>
<li> list item  1</li>
<li> list item 2 </li>
</ul>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        </nav>
+        <hr>
+        <main>
+            <section id="about">
+                <h2>About the City</h2>
+                <a href="">Back to the top</a>
```
Comment (mpagels) [inline index.html]: the href is empty? did you forgot to link to the top ;-)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
...m2qKZibDJQHqsUst+WI5v2A/nrlWWVKL+X7HZLDFOU0ur/AHPP4ehDsDcMxCuHIVVj4xP++VSb+FDH+w1YJZlh2uzd5XUdtjSyBA1stFgJFd6+FV+Cb12YiPZbSGie7C/IIAH6PIqz+r5A49/a69J6ZF2YY3USKdstq3KMD5tT7+4N8avJNqnL3f7mMEknr6pfsyo9W+rFkL1DuCLa/SuDEr5KKQSt8/xEAHxdCs9P2z7xkB3T5cIFs2q8KMaBpfV+373r0Yf4d7aFhuYpZ0vtnth0KWzov8aMSPVdEk8eR7VXcfS6bOeQq5Zvwu6lUi0xaJfFA8qfj/8AeqhmxviJg4Sd2WHp8O0G8/Auu33cTU6u4R3/AMsv6WWvgDjj+p1mx+n4rQqsasx8Jttv/qANMQSou/k/fjW/8N+hxz+ub1OoBDABTxHx49wPcV76usyCPsMvl54o+bICs+Te/mhj9rv2rXJmlPeos6YOEI/mXNAX/pmMiooTER16o8mruE1fbPFeBfB5oVelH1l0pn/CAm0A3K+Kti6YAWaHpVvJHj+l43mxilouit6VNMqt7WPI1X/8T+tr0/bpue13SSYgpfFRmGaz6TdV4FfvrHHGUpUu0CzpOLZUN70yBGSTcJHHEgxyYEg+1uppbN1xwaHnTTbdU2zxhEkjZxEthJAWDB6OaqbPx7gWPOlC7vd9V2kM3fXbgGS44oziWVmAIJewP9pJF86o3SfqF9tPKGBktHjyDtG2QJxdiLyIrwfPHPGumGJtNN20aSz9OqTPSP8AD76d7uw27jG+7yCqkgK9cekkGvuPA++o/XP0/L+P2zIzGMHb5ooyFCVizMKPAX3IPkcc6Q9I/wAR96qfk9pRdESJnZ5JIxwA9h49r86YxfX88cRkaNJpsmDSSXRBc2FVaxGIxoH2U81RJQnGWy75I2crj3EsG5SLMCKCKU05iVhgCb5X9Cqoq28D9J44vVV+vPxUMDCeKCOLNY1RGyXLLP0+gFAygrkCCK9rJ0H1n/EmSSbbPHt4okjOZjssHJPORAHFeBXnnnxpd9aF9xBt9w7t+Zn+WAqxpicTiFUeSMufGRrTxxcZJyJm201FFS2XUCqBTHnXg2w4+ONZrru+mhGxu+FN1XkA/Os127pnLrJcH//Z" alt="picture of the city" height="90px">
```
Comment (mpagels) [inline index.html]: Is there a better way to implement the picture? this is a lot of noise code. ;-)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                <p>
+                    Belo Horizonte (Beautiful Horizon) is the sixth-largest city in Brazil, with a population around 2.7 million and with a metropolitan area of 12.7 million people. It is the 13th-largest city in South America and the 18th-largest in the Americas. The metropolis is anchor to the Belo Horizonte metropolitan area, ranked as the third-most populous metropolitan area in Brazil and the 17th-most populous in the Americas. Belo Horizonte is the capital of the state of Minas Gerais, Brazil's second-most populous state. It is the first planned modern city in Brazil. 
+                </p>
+            </section>
+            <hr>
+            <section id="history">
+                <h2>History</h2>
+                <a href="">Back to the top</a>
```
Comment (mpagels) [inline index.html]: same missing href

File: index.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                <img src="https://i.pinimg.com/originals/e5/74/2c/e5742cd99e3266d08c5e968036e283e0.jpg" alt="" height="160px">
```
Comment (mpagels) [inline index.html]: way better solution to add a image to the website ;-)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+                    Another important factor contributing to the growth of the village was people immigrating from the São Francisco River region, who had to pass through Curral d'el Rey to reach southern parts of Brazil. Travelers usually visited a small wooden chapel, where they prayed for a safe trip, so the chapel was named Capela da Nossa Senhora da Boa Viagem, which means "Chapel of Our Lady of the Good Journey." After the construction of Belo Horizonte, the old baroque chapel was replaced by a neo-gothic church that became the city's cathedral. 
+                </p>
+            </section>
+            <hr>
+            <section id="location">
+                <h2>Location</h2>
+                <a href="">Back to the top</a>
```
Comment (mpagels) [inline index.html]: ;-)

**Review by ManuelRemer — APPROVED** [review]
Looks very good. Good idea to leave your old stuff as a comment?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+                <p>
+                    The regional Minas Gerais' food and the now internationally known drink of cachaça are very popular and highly rated in the capital. Belo Horizonte is internationally known as the "capital of neighborhood bars."
+                </p>
+            </section>
+            <hr>
+            <section id="more">
+                <h2 >More</h2>
+                <p><a href="https://en.wikipedia.org/wiki/Category:Football_clubs_in_Belo_Horizonte">Soccer </a> ⚽️</p>
+                <p><a href="https://www.artchitectours.com/tour/belo-horizonte-modernism/">Architecture </a> 🌇</p>
+                <p><a href="https://www.brazilnaturetours.com/brazil-tours/belo-horizonte">Nature </a> 🌲</p>
```
Comment (mpagels) [inline index.html]: could this be a list?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+                </ul>
+            </section>
+            <section>
+                <h2>Social Media Links</h2>
+                <a href="">Facebook</a>
+                <a href="">LinkedIn</a>
+                <a href="">Twitter</a>
+            </section>
+        </main>
+        <footer><em>made by Eduardo</em> ⚽️</footer> -->
```
Comment (mpagels) [inline index.html]: comments!. You can delete this because this will be also in your served html file.

**Review by mpagels — CHANGES_REQUESTED** [review]
Awesome job eduardo. Found some things. Please review my review. :-)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+    </body>
+</html>
```
Comment (yvlange) [inline index.html:85]: I like the 'back to the top' thingy. Excited to see your hometown :)

File: index.html (hunk 1 above)
Comment (eduardoccmelo) [reply index.html]: got it

File: index.html (hunk 2 above)
Comment (eduardoccmelo) [reply index.html]: I included an id on the header, then the href now links to the top of the page. When it is empty, I think it just reloads the page right?

Comment (mpagels) [reply index.html]: When it's empty it should do nothing.

File: index.html (hunk 3 above)
Comment (eduardoccmelo) [reply index.html]: I know 🙈 I just added everything locally now :)

File: index.html (hunk 7 above)
Comment (eduardoccmelo) [reply index.html]: 🤙🏼

File: index.html (hunk 2 above)
Comment (eduardoccmelo) [reply index.html]: ok

**Review by mpagels — APPROVED** [review]
Very good!

File: index.html (hunk 9 above)
Comment (eduardoccmelo) [reply index.html:85]: Kind of practical when you don´t need to scroll all the way up :D 

Thanks!!!!

**Pull request merged**